src/App.js

```diff
@@ -1,7 +1,8 @@
 
 function App() {
   return (<>
-    <h1>LETS GET STARTED!</h1>
+    <h1>LETS MAKE PROJECT!</h1>
```

Comment:
you need use text-transform in CSS

try

`h1 {
  text-transform: uppercase;
}`

Reply:
Allow only one h1 tag on page

Reply:
It wasn`t written in TT)

Reply:
but i will add it

Reply:
fixed